tests/unit/tasks/server/express-server-test.js

```diff
@@ -228,6 +229,43 @@ describe('express-server', function () {
         });
     });
 
+    // TODO: this is not supported by supertest https://github.com/visionmedia/supertest/pull/494 should fix that
```

Comment:
We might want to consider a switch, that PR is pretty old and doesn't really have any traction as far as I can tell...

Reply:
will follow up with the pr to see what is happening there.

Reply:
followed up here https://github.com/visionmedia/supertest/pull/691.